projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -878,7 +878,7 @@ public function grunion_source_filter_results( $query ) {
 	public function grunion_manage_post_row_actions( $actions ) {
 		global $post;
 
-		if ( 'feedback' !== $post->post_type ) {
+		if ( ! ( is_object( $post ) && isset( $post->post_type ) && 'feedback' === $post->post_type ) ) {
```

Comment:
You could do this:

```suggestion
		if ( ! $post instanceof \WP_Post || 'feedback' !== $post->post_type ) ) {
```

Reply:
Yep, I suppose that's more precise. Will go ahead and adopt that.

Reply:
Oops, sorry! More precise but with a built-in error. 😄